package.json

```
@@ -37,15 +38,15 @@
     "refresh:config": "test -n $BASH_VERSION && bash ./tasks/chromatic-config-creation.sh || exit 0",
     "refresh:directory": "test -n $BASH_VERSION && bash ./tasks/clean-up-after-migration.sh || exit 0",
     "refresh:env": "test -n $BASH_VERSION && bash ./tasks/copy-env-from-root.sh || exit 0",
-    "release": "cross-env NODE_ENV=production nx reset && yarn builder tag:component,ui-icons && lerna publish --no-private",
-    "release:beta-from-package": "yarn release from-package --conventional-prerelease --preid beta --pre-dist-tag beta",
+    "release": "cross-env NODE_ENV=production nx reset && yarn builder tag:component,ui-icons && changeset publish",
```

```suggestion
    "release": "cross-env NODE_ENV=production nx reset && yarn builder token,tag:component,ui-icons && changeset publish",
```

Let's include tokens here too in case it gets dropped from being built by the components to prevent publishing without the compiled token output.